docusaurus/docs/protocol/primitives/probabilistic_proofs.md

```diff
@@ -1,23 +1,514 @@
+
+This represents the Supplier's earnings before the penalty is applied.
+
+If the Supplier chooses to leave the network at this point in time, they will
+have successfully gamed the system.
+
+#### Expected Penalty: Slashing amount for Dishonest Supplier
+
+The penalty is a fixed amount `S` when caught.
+
+#### Total Profit: Expected Value Calculation for Dishonest Supplier AFTER Penalty
+
+$$ E[\text{Total Profit}] = E[\text{Total Rewards}] - S = R \cdot \frac{q}{p} - S $$
+
+### Honest Supplier: Calculating the Expected Value
+
+- **Expected Rewards per Claim**: $$ E[\text{Reward per Claim}] = R $$
+- **No Penalties**: Since the honest Supplier always provides valid Proofs when required, they avoid penalties.
+- **Expected Profit for Honest Supplier (1 Claim)**:
+
+  $$ E[\text{Total Profit}] = R $$
+
+### Setting Parameters to Deter Dishonest Behavior
+
+To deter dishonest Suppliers, we need:
+
+$$ E[\text{Total Profit}_{\text{Dishonest}}] \leq E[\text{Total Profit}_{\text{Honest}}] $$
+
+Substituting the expected values:
+
+$$ R \cdot \frac{q}{p} - S \leq R $$
+
+Since `q = 1 -p`, we can simplify the inequality to:
+
+$$ R \left( \frac{1 - 2p}{p} \right) \leq S $$
+
+#### Solving for Penalty `S`
+
+However, since `p` is between 0 and 1, `1 - 2p` can be negative if `p > 0.5`.
+To ensure `S` is positive, we consider `p ≤ 0.5`.
+
+Alternatively, to make the penalty effective, we can set:
+
+$$ S = R \cdot \left( \frac{1 - p}{p} \right) $$
+
+This ensures that the expected profit for dishonest Suppliers is `0` or negative:
+
+$$ E[\text{Total Profit}_{\text{Dishonest}}] = R \cdot \frac{q}{p} - S = R \cdot \frac{q}{p} - R \cdot \frac{q}{p} = 0 $$
+
+### Example Calculation
+
+Assume:
+
+- Reward Per Claim: `R = 10`
+- ProofRequestProbability: `p = 0.2`
+- `q = 0.8`
+
+Calculate the expected profit for a dishonest Supplier:
+
+1. **Expected Number of False Claims Before Getting Caught**:
+
+   $$ E[K] = \frac{q}{p} = \frac{0.8}{0.2} = 4 $$
+
+2. **Expected Total Rewards**:
+
+   $$ E[\text{Total Rewards}] = R \cdot E[K] = 10 \cdot 4 = 40 $$
+
+3. **Penalty**:
+
+   $$ S = R \cdot \left( \frac{1 - p}{p} \right) = 10 \cdot \left( \frac{0.8}{0.2} \right) = 40 $$
+
+4. **Expected Profit**:
+
+   $$ E[\text{Total Profit}] = E[\text{Total Rewards}] - S = 40 - 40 = 0 $$
+
+The dishonest Supplier has an expected profit of `0`, making dishonest behavior unattractive compared to honest behavior, which yields a profit of `R = 10` units per Claim without risk of penalty.
+
+### Generalizing the Penalty Formula
+
+To ensure that dishonest Suppliers have no incentive to cheat, set the penalty `S` such that:
+
+$$ S = R \cdot \frac{q}{p} = R \cdot \left( \frac{1 - p}{p} \right) $$
+
+This makes the expected profit for dishonest behavior `0`:
+
+$$ E[\text{Total Profit}_{\text{Dishonest}}] = R \cdot \frac{q}{p} - S = 0 $$
+
+### Considering false Claim Variance
+
+While the expected profit is `0`, the variance in the number of successful false
```

Comment:
Variance works both ways, hence my previous comment. This can be a justification for changing the calculation of `S` as I suggest. This is because the attacker might not be caught until much later and result in a net profit.

Reply:
My gut + intuition is telling me you're right, but I'm still forming a mental model of the underlying math to embody understanding it truly.

Will follow up in the other thread.